modules/nw-ovn-ipsec-verification.adoc

```diff
@@ -10,30 +10,29 @@ As a cluster administrator, you can verify that IPsec is enabled.
 
 .Verification
 
-. To find the names of the OVN-Kubernetes control plane pods, enter the following command:
+. To find the names of the OVN-Kubernetes data plane pods, enter the following command:
 +
 [source,terminal]
 ----
-$ oc get pods -n openshift-ovn-kubernetes | grep ovnkube-master
+$ oc get pods -n openshift-ovn-kubernetes | grep ovnkube-node
```

Comment:
Discussed and updated based on conversation with @anuragthehatter via slack. `nbdb` is now on data plane pods not on control plane so the procedural step needed to change.

Reply:
Thanks @JoeAldinger So grep seems like a old school way. May be we can do following. Bascially pods with label `app=ovnkube-node`
```
oc get pods -n openshift-ovn-kubernetes -l=app=ovnkube-node
```

Reply:
rest lgtm. thanks